### DISCUSSION TITLE
How to pass data object along with  the new resetForm json response

### DISCUSSION THREAD

**Discussion by taylrd** [edmundhung/conform#425] [Q&A] [answered]
Hi, I love the new API's! Im upgrading my  Remix codebase to V1. I have a form on a page that needs to be reset after successful completion and this works great:

```
return json({ result: submission.reply({ resetForm: true }) })
```

However, I need to provide a better user experience and pass along the last submission data to let the client know that, in this case, the student name was successfully added to their roster.  

Is there  a way to pass along more information with this implementation?

**Comment by edmundhung** [comment, marked as answer]
When you set the `resetForm` option, Conform will only includes the minimal data needed for the form reset and avoid sending data that's unnecessary. If you wanna include additional info, you can include them manually in the action response:

```tsx
return json({
  result: submission.reply({ resetForm: true }),
  submittedValue: submission.value, 
});
```

**Comment by taylrd** [comment]
Thanks Edmund, that worked. I was expecting a status to come with the result response and that was what was getting me.
